File: Beecrowd/Beginner/1101 - 1190/1134 - Type of Fuel/C++/1134_TypeofFuel.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int alcohol = 0, gasoline = 0, diesel = 0;

    while (true) {
        int code;
        cin >> code;

        if (code == 1) alcohol++;
        else if (code == 2) gasoline++;
        else if (code == 3) diesel++;
        else if (code == 4) break;
    }

    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << alcohol << endl;
    cout << "Gasolina: " << gasoline << endl;
    cout << "Diesel: " << diesel << endl;
    
    return 0;
}

File: Beecrowd/1 - Beginner/1101 - 1190/1134 - Type of Fuel/1134_TypeofFuel.py
alcohol = 0
gasoline = 0
diesel = 0

while True:
    code = int(input())

    if code == 1:
        alcohol += 1
    elif code == 2:
        gasoline += 1
    elif code == 3:
        diesel += 1
    elif code == 4:
        break

print("MUITO OBRIGADO")
print(f"Alcool: {alcohol}")
print(f"Gasolina: {gasoline}")
print(f"Diesel: {diesel}")
